.github/package-crux-mir.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+DATE=`date "+%Y-%m-%d"`
+PKG=crux-mir-$DATE
+EXE=`cabal v2-exec which crux-mir | tail -1`
+
+rm -rf $PKG
+
+mkdir $PKG
+mkdir $PKG/bin
+mkdir $PKG/doc
+
+cabal v2-build exe:crux-mir
+cp $EXE $PKG/bin
+cp `which cargo-mir-json` $PKG/bin
+cp `which mir-json` $PKG/bin
+cp `which mir-json-callgraph` $PKG/bin
+cp `which mir-json-dce` $PKG/bin
+cp `which mir-json-rustc-wrapper` $PKG/bin
+cp crux-mir/README.md $PKG/doc
+cp -r crux-mir/rlibs $PKG
+
+tar cf $PKG.tar.gz $PKG
```

Comment:
Should be `tar czf`, to enable gzip compression.  (I was wondering how on earth github managed to fit a 300MB .tar.gz into a 40MB .zip file...)

Reply:
Good catch! I should fix that for `crux-llvm`, too.